charts/coturn/templates/issuer.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if and .Values.tls.enabled .Values.tls.certManager.enabled .Values.tls.certManager.issuer.create }}
```

Comment:
Please drop this file, for the reasons described in the other comments.

Reply:
Done in [3cae51b](https://github.com/wireapp/wire-server/pull/2620/commits/3cae51b9080c3c316e57fa93fe396418c348f4c9)